keel-slack/src/main/kotlin/com/netflix/spinnaker/keel/slack/callbacks/ManualJudgmentCallbackHandler.kt

```diff
@@ -63,13 +64,25 @@ class ManualJudgmentCallbackHandler(
    * For example, if user gyardeni approved the notification, this function will add:
    * "@Gal Yardeni hit approve on 2021-02-12 11:05:57 AM" and will marked the original text with strikethrough.
    */
-  fun updateManualJudgementNotification(response: BlockActionPayload): List<LayoutBlock> {
+  fun oldUpdateManualJudgementNotification(response: BlockActionPayload): List<LayoutBlock> {
```

Comment:
Why the rename?

Reply:
arg oops, good catch. it was an accident from my other idea. I'll fix